test/fixtures/workload/fibonacci.js

```diff
@@ -4,5 +4,15 @@ function fib(n) {
   return fib(n - 1) + fib(n - 2);
 }
 
-const n = parseInt(process.env.FIB) || 35;
+let FIB = 30;
+// This is based on emperial values - in the CI, on Windows the program
+// tend to finish too fast then we won't be able to see the profiled script
+// in the samples, so we need to bump the values a bit. On slower platforms
+// like the Pis it could take more time to complete, we need to use a
+// smaller value so the test would not time out.
+if (common.isWindows) {
+  FIB = 40;
+}
+
+const n = parseInt(process.env.FIB) || FIB;
```

Comment:
Here's where the FIB environment variable overrides the default (40 on Windows, 30 everywhere else).